locale/en.po

```diff
@@ -5233,14 +5258,18 @@ msgstr "Protect just created species from species cap"
 msgid "PROTEINS"
 msgstr "Proteins"
 
-#: ../simulation_parameters/microbe_stage/organelles.json:134
+#: ../simulation_parameters/microbe_stage/organelles.json:154
 msgid "PROTOPLASM"
 msgstr "Protoplasm"
 
 #: ../src/gui_common/CreditsScroll.cs:530
 msgid "PULL_REQUESTS_PROGRAMMING"
 msgstr "Pull Requests / Programming"
 
+#: ../src/microbe_stage/organelle_unlocks/UndiscoveredOrganellesButton.tscn:93
+msgid "QUESTION"
```

Comment:
Note to self: this is too unclear translation key which I should fix before merging.